plugin/audit.go

```diff
@@ -24,29 +24,27 @@ import (
 type GeneralEvent byte
 
 const (
-	// Log presents log event.
-	Log GeneralEvent = iota
-	// Error presents error event.
+	// Starting represents a GeneralEvent that is about to start
+	Starting GeneralEvent = iota
```

Comment:
I think you can keep `Log` for backwards compatibility along with `Result` and `Status` (any that are actually in use) or make a new enumeration.

Reply:
The motivation for changing this is covered in the issue description:
> Log itself is ambiguous because it doesn't say if this means the event is complete, or is about to start. I thought it would be OK to rename, since it is already going to break the existing audit plugin by the next change

I also confirmed that both internal code and plugins do not depend on these.

Reply:
There are also no events generated for `Result` or `Status`. The only event type currently used is `Log`.

Reply:
@lysu confirmed these were not used and renaming is OK, so I am going to resolve this.